packages/mjml-group/README.md

```diff
@@ -61,5 +61,5 @@ attribute           | unit        | description                    | default att
 width               | percent/px  | group width                    | (100 / number of non-raw elements in section)%
 vertical-align      | string      | middle/top/bottom              | top
 background-color    | string      | background color for a group   | n/a
-direction           | ltr / rtl   | set the display order of direct children | ltr
```

Comment:
this should stay too

Reply:
Added the `9d` version of `direction` to `next`.

Reply:
ok for the `direction` doc changes 👍 it's fine like this indeed